code/modules/jobs/job_types/head_of_security.dm

```diff
@@ -74,14 +74,14 @@
 	suit_store = /obj/item/tank/internals/oxygen
 	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/gun/energy/e_gun=1, /obj/item/ammo_box/magazine/co9mm=1) //WS edit - free lethals
 
-/datum/outfit/job/hos/sergeant
+/datum/outfit/job/hos/syndicate
 	name = "Sergeant (Head of Security)"
 
-	uniform = /obj/item/clothing/under/syndicate/gorlex
-	head = /obj/item/clothing/head/HoS/beret/syndicate
+	uniform = /obj/item/clothing/under/syndicate/combat
+	head = /obj/item/clothing/head/warden
 	suit = /obj/item/clothing/suit/armor/vest/syndie
-	r_pocket = /obj/item/melee/transforming/energy/sword
-	backpack_contents = list(/obj/item/melee/baton/loaded=1)
+	id = /obj/item/card/id/syndicate_command/operative
```

Comment:
replace this with /obj/item/card/id/syndicate_command/crew_id , the operative id doesn't actually display the character's real name. Additionally, make sure every job has this type of id if it doesn't inherit a syndie crew or captain id from its parent type.